src/auth/auth.styles.js

```diff
@@ -17,6 +17,10 @@ export const SignUpStyles = styled.div`
   align-items: center;
 `;
 
+export const SignUpGreetingStyles = styled.div`
+  margin-top: 4rem;
```

Comment:
Why do we need this top margin here? For the centering?

Reply:
It would be also good to stick to a unit, we don't need `rems` here.

Reply:
If we put a height of `620px` to the AvatarsStyles it is enough, oder? And that way we let the vertical centering to be done by flex or/and grid.

Reply:
did u try without margin? 
this margin is the distance on the signup up sheet to the nav - its on top so not for centering.
which unit , do you mean px?
this and the avatars are different styles

Reply:
> which unit , do you mean px?

Yes, we should not use `pxs` and `rems`, unless we have a rule for in. In this case we have used pxs for margins.

Reply:
> this and the avatars are different styles

Your are right, I mixed the components names when writing, my bad.